### DISCUSSION TITLE
How to deploy a Remix web app to Cloudflare?

### DISCUSSION THREAD

**Discussion by matrixersp** [vercel/turborepo#6816] [Help] [answered]
I'm trying to deploy a Remix app through GitHub Actions, which I created following Cloudflare's documentation (https://developers.cloudflare.com/pages/framework-guides/deploy-a-remix-site/) using Bun.
When deployed as a standalone app it deploys successfully, but when I add it to my turborepo and deploy it, a url get generated that returns a `No webpage found` error. Do I need to make any configuration changes or additions to either the Turborepo itself or the Remix app to make it deploy successfully?

**Comment by jahilliard** [comment]
Any movement here?

**Reply by anthonyshew** [reply]
https://github.com/vercel/turbo/discussions/6816#discussioncomment-9257360

**Comment by anthonyshew** [comment]
This sounds like a missing or misconfigured [`outputs` key](https://turbo.build/repo/docs/reference/configuration#outputs). If the assets required to run the application are not included in the `outputs` key, Turborepo will not restore the build outputs from cache.

**Reply by jahilliard** [reply, marked as answer]
Im pretty sure it's not `outputs` key. 

I was able to solve it for my use case. It's not a turborepo related issue. It's likely that people experiencing this error are trying to launch a remix app that's not migrated to Vite, which is why everything builds correctly in Cloudflare and it seems "deployed", but it's not found. It's cause cf pages only works with remix on Vite. 

TLDR migrate your app to Vite and it should work. 

(Adding in case someone stumbles on this issue confused, like I was)
